### PR TITLE
Added more events

### PR DESCRIPTION
**Pull request by brantje**
### Short Description:

Add more events to the bot

### CODE REVIEW

**Comment by kazyka** [comment]
Looks fine

:+1:

File: pokemongo_bot/cell_workers/seen_fort_worker.py [hunk 1]
```diff
@@ -58,13 +61,15 @@ def work(self):
...
                                "green")
+                    rewards.append(str(item_count) + "x " + item_name)
                 if self.config.recycle_items:
                     recycle_worker = RecycleItemsWorker(self.bot)
                     recycle_worker.work()
 
             else:
                 logger.log("[#] Nothing found.", "yellow")
 
+            self.bot.fire('fort_spinned', rewards=rewards)
```
Comment (epicblood) [inline pokemongo_bot/cell_workers/seen_fort_worker.py:72]: maybe make this an object instead of a string?
like `{'xp':50, 'pokeball':3, 'greatball':1}`

Comment (brantje) [inline pokemongo_bot/cell_workers/seen_fort_worker.py:72]: Fixed it.

**Comment by meyer9** [comment]
:+1: merge conflicts

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/103/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by MaxLeiter** [comment]
:+1: fix the conflicts and good to go :) thanks!

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/103/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by kazyka** [comment]
@wchill can you fix the conflicts so we can merge?

**Comment by MaxLeiter** [comment]
@brantje needs to fix the conflicts, @kazyka

**Comment by kazyka** [comment]
@MaxLeiter had many tabs open, thought it was wchill :O my mistake :(

**Comment by MaxLeiter** [comment]
Sorry for the delay; could you please rebase this so we can test on master?

**Comment by tehp** [comment]
@brantje

**Comment by wchill** [comment]
Closing as no progress has been made by brantje.

**Pull request closed without merging**